diff --git a/build/client/run.sh b/build/client/run.sh
index e93c171..650c27d 100755
--- a/build/client/run.sh
+++ b/build/client/run.sh
@@ -100,7 +100,7 @@ dfget-go() {
     echo "BUILD: dfget-go"
     test -f ${BIN_DIR}/${DFGET_BINARY_NAME} && rm -f ${BIN_DIR}/${DFGET_BINARY_NAME}
     cd ${BUILD_SOURCE_HOME}/cmd/dfget
-    go build -o ${BIN_DIR}/${DFGET_BINARY_NAME}
+    CC=/usr/bin/musl-gcc go build --ldflags '-linkmode external -extldflags "-static"' -o ${BIN_DIR}/${DFGET_BINARY_NAME}
     chmod a+x ${BIN_DIR}/${DFGET_BINARY_NAME}
 }
